cmd/grype/cli/options/fix_channels.go

```diff
@@ -0,0 +1,51 @@
+package options
+
+import (
+	"strings"
+
+	"github.com/anchore/clio"
+	"github.com/anchore/grype/grype/distro"
+)
+
+type FixChannelEnabled string
+
+type FixChannels struct {
+	// EUS is the Extended Update Support channel for RHEL
+	RedHatEUS FixChannel `yaml:"redhat-eus" json:"redhat-eus" mapstructure:"redhat-eus"`
+}
+
+type FixChannel struct {
+	IDs      []string                 `yaml:"ids" json:"ids" mapstructure:"ids"`                // the list of distro release IDs that this channel applies to
+	Apply    distro.FixChannelEnabled `yaml:"apply" json:"apply" mapstructure:"apply"`          // whether this channel should be applied to the distro when matching
+	Versions string                   `yaml:"versions" json:"versions" mapstructure:"versions"` // the version constraint that this channel applies to, e.g. ">= 8.0"
+}
```

Comment:
note to self: we should probably remove IDs and Versions from the app config -- these will be lower level API configurations, but not available to the CLI. This way we have a chance to make this more data driven later

Reply:
I've removed IDs and Versions for now from the user config, but it still remains in the API config